.pre-commit-config.yaml

```
@@ -92,3 +92,7 @@ repos:
           - types-tqdm
           - typing-extensions
           - wandb
+  - repo: https://github.com/Yelp/detect-secrets
+    rev: v1.4.0 # Use the latest version
```

```suggestion
    rev: v1.5.0 # Use the latest version
```

You have a comment stating this, but then 1.4 is not the latest version 😆